src/SFA.DAS.Commitments.Application/Services/V2EventsPublisher.cs

```diff
@@ -42,5 +44,40 @@ public async Task PublishApprenticeshipDeleted(Commitment commitment, Apprentice
                 _logger.Error(e, $"{logMessage} failed");
             }
         }
+
+        public async Task PublishApprenticeshipCreated(IApprenticeshipEvent apprenticeshipEvent)
+        {
+            var logMessage = $"Publish ApprenticeshipCreatedEvent message. Provider:{apprenticeshipEvent.Commitment.ProviderId} Commitment:{apprenticeshipEvent.Commitment.Id} Apprenticeship:{apprenticeshipEvent.Apprenticeship.Id}";
+
+            try
+            {
+                var priceEpisodes = apprenticeshipEvent.Apprenticeship.PriceHistory
+                    .Select(x => new PriceEpisode { FromDate = x.FromDate, ToDate = x.ToDate, Cost = x.Cost}).ToArray();
+
+                await _endpointInstance.Publish<IApprenticeshipCreatedEvent>(ev =>
+                {
+                    ev.ApprenticeshipId = apprenticeshipEvent.Apprenticeship.Id;
+                    ev.CreatedOn = DateTime.Now; // Get this from somewhere
```

Comment:
Should be UtcNow

Reply:
Should this be using the ICurrentDateTime service rather than use UtcNow directly?